app/interactors/update_project/save_record.rb

```diff
@@ -0,0 +1,18 @@
+class UpdateProject
+  class SaveRecord
+    include Interactor
+
+    delegate :prepared_project_params, to: :context
+
+    def call
+      context.project = project
+      context.fail!(error: "Invalid data") unless project.save
+    end
+
+    private
+
+    def project
+      @project ||= Project.new(prepared_project_params)
```

Comment:
do you create new project in update_project interactor?

Reply:
Is your code runable?
Can you rerequest the review when all will work correct

Reply:
Fixed